Tests/AblyChatTests/DefaultMessagesTests.swift

```diff
@@ -10,7 +10,7 @@ struct DefaultMessagesTests {
         // Given
         let realtime = MockRealtime()
         let chatAPI = ChatAPI(realtime: realtime)
-        let channel = MockRealtimeChannel()
+        let channel = MockRealtimeChannel(attachResult: .success)
```

Comment:
The tests don't attach the channel so it's a bit noisy for them to have to specify `attachResult`; can we revert this?